### ISSUE TITLE
Can you add touch screen support?

### ISSUE THREAD

**Issue by duxiangjiang** [profzei/Matebook-X-Pro-2018#133]
I think the touch screen is ok in some previous releases, but not work now(i use the latest release MateBook X Pro EFI v1.9.0).

**Comment by profzei** [comment]
Hi @duxiangjiang 
Both release v. 1.8.0 and v. 1.9.0 do not have touchscreen support by default. Why?
As reported (only partially) in the [Changelog](https://github.com/profzei/Matebook-X-Pro-2018/blob/master/Changelog.md), it seems that all `VoodooI2C` versions after 2.4 are affected to a greater or lesser extent by `Multi_GPI0` issues.
The solution proposed so far by `VoodooI2C` developers is only a workaround since they do not really solve the problem: proposing to use the GPI0 pinning for one device and polling method for the other is not a fix because it increases interrupts and this degrades battery life.
With their current `VoodooI2C` release v. 2.6.3 they say they have finally solved (understood) the problem related to the loss of the touchpad functionality after sleep... I have not yet checked... I think it is true since the used approach is (obviously) a generalization of the one I used here in my repository for the Matebook X Pro over a month ago.
This being the case and not needing to use the touchscreen in my daily work, for the moment I have disabled this function  pending improvements that can only come from `VoodooI2C` developers without their work we would not even be here!
Making both the touchpad and the touchscreen work in `Multi_GPI0` mode will certainly be possible since it worked before `VoodooI2C` v. 2.4... then they decided to rewrite many perhaps too many pieces of code to significantly improve the gestures smoothness...
